pynamodb/models.py

```diff
@@ -442,15 +466,20 @@ def update(self, attributes=None, actions=None, condition=None, conditional_oper
 
         kwargs.update(condition=condition)
         kwargs.update(actions=actions)
-        data = self._get_connection().update_item(*args, **kwargs)
-        for name, value in data[ATTRIBUTES].items():
-            attr_name = self._dynamo_to_python_attr(name)
-            attr = self.get_attributes().get(attr_name)
-            if attr:
-                setattr(self, attr_name, attr.deserialize(attr.get_value(value)))
-        return data
 
-    def save(self, condition=None, conditional_operator=None, **expected_values):
+        if in_transaction is None:
+            data = self._get_connection().update_item(*args, **kwargs)
+            for name, value in data[ATTRIBUTES].items():
+                attr_name = self._dynamo_to_python_attr(name)
+                attr = self.get_attributes().get(attr_name)
+                if attr:
+                    setattr(self, attr_name, attr.deserialize(attr.get_value(value)))
+            return data
+
+        operation_kwargs = self._get_connection().get_operation_kwargs_for_update_item(*args, **kwargs)
+        in_transaction.add_update_item(operation_kwargs)
```

Comment:
no return here, unlike the others? 🤔

Reply:
I'm still trying to figure out the best way for handling mypy issues that come with the fact that transactions don't return anything